capstone/files/enrichment/virustotal.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python
+# Copyright (C) 2015-2020, Wazuh Inc.
+# October 19, 2017.
+#
+# This program is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+# Wazuh, Inc <support@wazuh.com>
+
+import json
+import sys
+import time
+import os
+from socket import socket, AF_UNIX, SOCK_DGRAM
+
+try:
+    import requests
+    from requests.auth import HTTPBasicAuth
+except Exception as e:
+    print("No module 'requests' found. Install: pip install requests")
+    sys.exit(1)
+
+try:
+    import vt
+except Exception as e:
+    print("No module 'vt' found. Install pip install vt-py")
+    sys.exit(1)
+
+# ossec.conf configuration:
+#  <integration>
+#      <name>virustotal</name>
+#      <api_key>api_key_here</api_key>
+#      <group>syscheck</group>
+#      <alert_format>json</alert_format>
+#  </integration>
+
+# Global vars
+
+DEBUG = False
+PWD = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+json_alert = {}
+now = time.strftime("%a %b %d %H:%M:%S %Z %Y")
+
+# Set paths
+LOG_FILE_PATH = '{0}/logs/integrations.log'.format(PWD)
+SOCKET_ADDR = '{0}/queue/ossec/queue'.format(PWD)
+
+
+def main(args):
+    debug("# Starting")
+
+    # Read args
+    alert_file_location = args[1]
+    apikey = args[2]
+
+    debug("# API Key")
+    debug(apikey)
+
+    debug("# File location")
+    debug(alert_file_location)
+
+    # Load alert. Parse JSON object.
+    with open(alert_file_location) as alert_file:
+        json_alert = json.load(alert_file)
+    debug("# Processing alert")
+    debug(json_alert)
+
+    # Request VirusTotal info
+    msg = request_virustotal_info(json_alert, apikey)
+
+    # If positive match, send event to Wazuh Manager
+    if msg:
+        send_event(msg, json_alert["agent"])
+
+
+def debug(msg):
```

Comment:
This logger is very basic, this is a good chance to learn about real python loggers. You'd need to configure two logger handlers:
* A [`BaseRotating`](https://docs.python.org/3/library/logging.handlers.html#baserotatinghandler) one.
* A [`Stream`](https://docs.python.org/3/library/logging.handlers.html#streamhandler) which will only be enabled if the `DEBUG` flag is set.